grammars/html.cson

```diff
@@ -350,12 +350,16 @@
   'entities':
     'patterns': [
       {
-        'captures':
+        'begin': '(&)([a-zA-Z0-9]+|#[0-9]+|#x[0-9a-fA-F]+)'
+        'beginCaptures':
+          '1':
+            'name': 'punctuation.definition.entity.begin.html'
+          '2':
+            'name': 'entity.name.entity.other.html'
+        'end': '(;)'
```

Comment:
Parentheses aren't required around this: you can just match the 0th capture instead.